denops/@ddc-filters/converter_fuzzy.ts

```diff
@@ -53,7 +53,9 @@ export class Filter extends BaseFilter<Params> {
       return {
         ...item,
         highlights: [
-          ...(item.highlights ?? []),
+          ...(item.highlights ?? []).filter((hl) =>
+            hl.name !== "ddc_fuzzy_matched_character"
+          ),
```

Comment:
Would you  mind tell me the reason for this change ?

Reply:
Previous highlights may remain after deletion with \<BS>.

Reply:
Great. Thank you for pointing out. I got it.